File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int [] sols = new int[2];
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    sols[0] = i;
                    sols[1] = j;
                }
            }
        }
        return sols;
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
      hmap = {}
      for index, num in enumerate(nums):
        compliment = target - num
        if compliment in hmap:
            return [index, hmap[compliment]]
        hmap[num] = index  